src/components/Inputs/Input.types.ts

```diff
@@ -212,6 +211,17 @@ export interface InputProps<T>
         OcBaseProps<T>,
         'onChange' | 'onFocus' | 'onBlur' | 'onKeyDown'
     > {
+    /**
+     * The input icon alignment.
+     * @default TextInputIconAlign.Left
```

Comment:
I'm assuming this is the chevron... it looks so much better (and normal) on the right. UX wants the chevron on the left?

Reply:
UX want the chevron on the right.

Reply:
Agree with that. Looked very funny on the left. Can we make right that the default then? ... or is this icon not the chevron?